bin/cylc-report-timings

```diff
@@ -187,6 +188,10 @@ class TimingSummary(object):
 
         """
         import pandas as pd
+
+        # Make sure lines don't wrap!
```

Comment:
The comment is not quite right, the lines might get wrapped, but the content of columns doesn't get truncated

Reply:
Thanks.  Updated accordingly.